src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -4,10 +4,9 @@
 
 public class Ui {
 
-    private static Scanner in = new Scanner(System.in);
-
     public static String getCommand() {
-        return in.nextLine();
+        Scanner userInput = new Scanner(System.in);
```

Comment:
Is there a reason why a new scanner is being declared every time a command is taken in? I believe this works the same way as our previous approach right?

Reply:
Edited it! Thank you for spotting that